src/Examples/Advanced/CustomProperties/Hydrocarbon_Processing_example.ipynb

```diff
@@ -299,13 +345,59 @@
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.R101.control_volume.properties_out[0.0]._material_density_term[Vap,hexene]\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.R101.control_volume.properties_out[0.0]._material_density_term[Vap,heptene]\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.R101.control_volume.properties_out[0.0]._material_density_term[Vap,octene]\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.R101.control_volume.heat\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].enth_mol_phase[Liq], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].enth_mol_phase[Vap], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].mole_frac_comp\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_out[0.0].enth_mol_phase[Liq], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_out[0.0].enth_mol_phase[Vap], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_out[0.0].mole_frac_comp\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.heat\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_in[0.0].enth_mol_phase[Liq], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_in[0.0].enth_mol_phase[Vap], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_in[0.0].mole_frac_comp\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_out[0.0].enth_mol_phase[Liq], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_out[0.0].enth_mol_phase[Vap], for _enthalpy_flow_term\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.properties_out[0.0].mole_frac_comp\n",
+      "2021-12-21 14:03:02 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.F101.control_volume.heat\n",
```

Comment:
See above re: the warnings

Reply:
These warnings are from my changes, and occur because `calculate_scaling_factors` is called on the entire model and only a small set of variables is scaled.